src/pages/nft/class/_classId/index.vue

```diff
@@ -1471,8 +1471,8 @@ export default {
       await this.handleCollectFromEdition(this.giftSelectedValue, giftInfo);
       this.isGiftDialogOpen = false;
     },
-    handleGiftFromEditionSelector(selectedValue) {
-      this.giftSelectedValue = selectedValue;
+    handleGiftFromEditionSelector() {
```

Comment:
@williamchong @AuroraHuang22  Should we also rename the func name & event name?

Reply:
yes go ahead